From 7bf1032167353cce32de01ed45dbbf2ea2a18823 Mon Sep 17 00:00:00 2001
From: Voon Weifeng <weifeng.voon@intel.com>
Date: Sat, 27 Jul 2019 07:53:45 +0800
Subject: [PATCH 41/52] REVERTME: net: stmmac: add EHL SGMII 2.5Gbps PCI info
 and PCI ID

Add EHL SGMII 2.5Gbps PCI info and PCI ID

Signed-off-by: Voon Weifeng <weifeng.voon@intel.com>
Signed-off-by: Ong Boon Leong <boon.leong.ong@intel.com>
---
 drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
index 4eb13a9..31e0608 100644
--- a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
+++ b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
@@ -726,6 +726,7 @@ static SIMPLE_DEV_PM_OPS(stmmac_pm_ops, stmmac_pci_suspend, stmmac_pci_resume);
 #define STMMAC_QUARK_ID  0x0937
 #define STMMAC_DEVICE_ID 0x1108
 #define STMMAC_EHL_SGMII1G_ID	0x4b31
+#define STMMAC_EHL_SGMII2G5_ID	0x4b32
 #define STMMAC_EHL_PSE0_RGMII1G_ID	0x4ba0
 #define STMMAC_EHL_PSE0_SGMII1G_ID	0x4ba1
 #define STMMAC_EHL_PSE0_SGMII2G5_ID	0x4ba2
@@ -744,6 +745,7 @@ static const struct pci_device_id stmmac_id_table[] = {
 	STMMAC_DEVICE(STMICRO, PCI_DEVICE_ID_STMICRO_MAC, stmmac_pci_info),
 	STMMAC_DEVICE(INTEL, STMMAC_QUARK_ID, quark_pci_info),
 	STMMAC_DEVICE(INTEL, STMMAC_EHL_SGMII1G_ID, ehl_sgmii1g_pci_info),
+	STMMAC_DEVICE(INTEL, STMMAC_EHL_SGMII2G5_ID, ehl_sgmii1g_pci_info),
 	STMMAC_DEVICE(INTEL, STMMAC_EHL_PSE0_RGMII1G_ID,
 		      ehl_pse0_rgmii1g_pci_info),
 	STMMAC_DEVICE(INTEL, STMMAC_EHL_PSE0_SGMII1G_ID,
-- 
2.7.4

